tests/test_taar_locale.py

```diff
@@ -1,10 +1,11 @@
 """Test suite for TAAR Locale Job."""
 
 import boto3
+import json
 import functools
 import pytest
 from moto import mock_s3
-from mozetl.taar import taar_locale
+from mozetl.taar import taar_locale, AMO_DUMP_BUCKET, AMO_DUMP_KEY
```

Comment:
`AMO_DUMP_BUCKET` and `AMO_DUMP_KEY` are actually in the scope of `taar_locale`. You can access them like the functions e.g. `taar_locale.AMO_DUMP_BUCKET`